tests/Index/SettingsTest.php

```diff
@@ -136,8 +137,7 @@ public function testDeleteAliasWithException(): void
             $indexAlias->delete();
             $this->fail('Should throw exception because you should delete the concrete index and not the alias');
         } catch (ClientResponseException $e) {
-            $response = ResponseConverter::toElastica($e->getResponse());
-            $error = $response->getFullError();
+            $error = \json_decode((string) $e->getResponse()->getBody(), true)['error']['root_cause'][0] ?? null;
```

Comment:
What is the reason you removed the ResponseCoverter here?

Reply:
`ResponseCoverter` in my opinion should work with code related to `src` folder  (production code). Official Elasticsearch client does not return `ResponseInterface`.

I'm planning to raise a new PR where we won't allow ResponseInterface as an option to `ResponseCoverter`

Reply:
Does this make migration from 7.x potentially harder? Lets discuss on the upcoming PR, code will make it much easier to fully understand what you mean by this.